ReserveSpot.Services.Api/Brokers/Loggings/LoggingBroker.cs

```diff
@@ -0,0 +1,35 @@
+﻿// ------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// ------------------------------------------------
+
+using Microsoft.Extensions.Logging;
+using System;
```

Comment:
Run code cleanup with the 'Place system directive on top' setting.

Reply:
Done code cleanup with the above option

Reply:
I still see the system is not sorted.